client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/index.jsx

```diff
@@ -203,6 +203,7 @@ class VisibleSubmissionEditIndex extends Component {
      * Therefore we need to manually touch all the fields
      */
     answers.forEach((answer = {}) => {
+      if (questions[answer.questionId].type !== 'VoiceResponse') return;
```

Comment:
```suggestion
      if (answer.questionId in questions && questions[answer.questionId].type !== 'VoiceResponse')  {
        return;
      }
```

Just in case the question ID key doesn't exist - we will `touch` it nevertheless.